sdk/eventhub/azure-eventhub/azure/eventhub/exceptions.py

```diff
@@ -12,33 +17,39 @@ class EventHubError(Exception):
     :vartype error: str
     :ivar details: The error details, if included in the
      service response.
-    :vartype details: dict[str, str]
+    :vartype details: list[str]
```

Comment:
is this correct ? below we are casting details to be `AmqpException`, but `self.details` is `Union[List, AmqpException]`